hail/python/hailtop/batch/batch.py

```diff
@@ -549,19 +554,23 @@ def write_output(self, resource: _resource.Resource, dest: str):
 
         Write a single job intermediate to a permanent location in GCS:
 
-        >>> b = Batch()
-        >>> j = b.new_job()
-        >>> j.command(f'echo "hello" > {j.ofile}')
-        >>> b.write_output(j.ofile, 'gs://mybucket/output/hello.txt')
-        >>> b.run()  # doctest: +SKIP
+        .. code-block:: python
+
+            b = Batch()
+            j = b.new_job()
+            j.command(f'echo "hello" > {j.ofile}')
+            b.write_output(j.ofile, 'gs://mybucket/output/hello.txt')
+            b.run()
```

Comment:
It's a shame to lose these doctests. Deferring cold storage validation to `b.run` would let us keep these.

Reply:
see above re: eagerness discussion